docs/pages/table.js

```diff
@@ -2718,3 +2552,39 @@ Checkboxes are often used in tables to allow for selecting and editing of multip
     </Page>
   );
 }
+
+export async function getStaticProps(): Promise<{|
+  props: {| generatedDocGen: {| [string]: DocGen |} |},
+|}> {
+  const docGen = await multipledocgen({
+    componentName: [
+      'Table',
+      'TableHeader',
+      'TableBody',
+      'TableFooter',
+      'TableCell',
+      'TableHeaderCell',
+      'TableSortableHeaderCell',
+      'TableRow',
+      'TableRowExpandable',
+    ],
+  });
+
+  docGen.Table.props.children.flowType.raw =
+    'React.ChildrenArray<React.Element< typeof Table.Body | typeof Table.Footer | typeof Table.Header >>';
```

Comment:
there's some extra whitespace within the "< >" in these types

Reply:
DONE